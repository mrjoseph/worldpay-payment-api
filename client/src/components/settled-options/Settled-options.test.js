import React from 'react';
import renderer from 'react-test-renderer';
import SettledOptions from './Settled-options';

describe('Header', () => {
  it('Should render the checkout view without props', () => {
    const rendered = renderer.create(<SettledOptions />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
