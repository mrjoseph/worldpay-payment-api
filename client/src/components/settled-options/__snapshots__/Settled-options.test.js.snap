// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render the checkout view without props 1`] = `
<div
  className="settled-payment-options"
>
  <h5>
    Refund options
  </h5>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque nec sagittis velit, vitae euismod est. Suspendisse at quam vitae elit rhoncus rhoncus ut at erat. In dapibus ipsum sodales varius pretium. Suspendisse malesuada placerat accumsan. Integer pulvinar felis ullamcorper turpis sollicitudin euismod
  </p>
  <button
    className="btn btn-primary"
    type="button"
  >
    Refund
  </button>
  <button
    className="btn btn-secondary"
    type="button"
  >
    Partial refund
  </button>
</div>
`;
